### PR TITLE
PMM-8077 Show operator version

### PR DESCRIPTION
**Pull request by tiagomotasantos**
**What this PR does / why we need it**: Shows the current version of the operator

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-8263

### CODE REVIEW

**Review by fabio-silva — APPROVED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

**Review by fabio-silva — DISMISSED** [review]

**Review by nicolalamacchia — DISMISSED** [review]

File: public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/UpdateOperatorModal/UpdateOperatorModal.styles.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ palette, spacing }: GrafanaTheme) => ({
+  modalWrapper: css`
+    div[data-qa='modal-body'] {
+      left: 27%;
```
Comment (atymchuk) [inline public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/UpdateOperatorModal/UpdateOperatorModal.styles.ts:7]: Does it render well on different screen sizes?

File: public/app/percona/dbaas/DBaaS.messages.tsx [hunk 2]
```diff
@@ -53,6 +54,23 @@ export const Messages = {
...
+      confirm: 'Update',
+      title: 'Confirm operator update',
+      buildUpdateOperatorMessage: (
+        operatorType: string,
+        newVersion: ReactNode,
+        kubernetesClusterName: ReactNode,
+        currentVersion?: string
+      ) => (
+        <>
+          Are you sure you want to update {operatorType} {currentVersion} to version {newVersion} in{' '}
```
Comment (fabio-silva) [inline public/app/percona/dbaas/DBaaS.messages.tsx:68]: Could we send this as a function to the messages file?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/UpdateOperatorModal/UpdateOperatorModal.styles.ts (hunk 1 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/components/Kubernetes/OperatorStatusItem/KubernetesOperatorStatus/UpdateOperatorModal/UpdateOperatorModal.styles.ts:7]: Screenshot 2021-09-03 at 10 09 19
Screenshot 2021-09-03 at 10 09 34
Screenshot 2021-09-03 at 10 09 50
Screenshot 2021-09-03 at 10 09 59

File: public/app/percona/dbaas/DBaaS.messages.tsx (hunk 2 above)
Comment (tiagomotasantos) [reply public/app/percona/dbaas/DBaaS.messages.tsx:68]: I'm not understanding the question @fabio-silva sorry. This is a function and this is the messages file 🤔

Comment (fabio-silva) [reply public/app/percona/dbaas/DBaaS.messages.tsx:68]: You're totally right. Sorry, let's pretend I never said this 😂

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**